api_tests/preprints/views/test_preprint_citations.py

```diff
@@ -128,15 +128,33 @@ def setUp(self):
         self.third_contrib.family_name = 'Schematics'
         self.third_contrib.save()
 
+        self.MLA_DATE_FORMAT = '%-d {month} %Y'
+        self.MLA_MONTH_MAP = {
```

Comment:
Can you just do the same formatting that the citeproc code is doing instead of reimplementing it here?

Reply:
Citeproc pulls this info from the MLA's csl file, [here](https://github.com/citation-style-language/styles/blob/master/modern-language-association.csl#L24-L38). I don't really think that's practical here.